core/templates/services/rte-helper.service.spec.ts

```diff
@@ -277,7 +277,7 @@ describe('Rte Helper Service', () => {
     var submitCallBackSpy = jasmine.createSpy('submit');
     var dismissCallBackSpy = jasmine.createSpy('dismiss');
     rteHelperService.openCustomizationModal(
-      [], {}, submitCallBackSpy, dismissCallBackSpy);
+      '', [], {}, submitCallBackSpy, dismissCallBackSpy);
```

Comment:
If this can't be empty in practice, then don't allow it to be empty. Pass in a proper ID.

Reply:
can I just pass in any id as a string in 3 cases? For example use
```
rteHelperService.openCustomizationModal(
      'video', [], {}, submitCallBackSpy, dismissCallBackSpy);
```
for all instead?

Reply:
Yes that's OK.